zql/docs/aggregate-functions/README.md

```diff
@@ -15,8 +15,7 @@ the events.
      + [`collect`](#collect)
      + [`count`](#count)
      + [`countdistinct`](#countdistinct)
```

Comment:
Unless #1892 is going to be addressed real soon, the doc  for countdistinct should probably get a note explaining it  may not yet work in Brim? ("may" because parallelism is only used  for archive stores)